radicle-cli/src/git.rs

```diff
@@ -204,6 +255,16 @@ pub fn rad_remote(repo: &Repository) -> anyhow::Result<(git2::Remote, Id)> {
     }
 }
 
+pub fn add_remote<'a>(
+    repo: &'a Repository,
+    name: &'a str,
+    url: &'a radicle::git::Url,
+) -> anyhow::Result<Remote<'a>> {
+    let url = url.to_string();
+    let remote = repo.remote(name, &url)?;
```

Comment:
nitpick: maybe it's just me, but I prefer not creating a new variable when it's something simple like this :)
```suggestion
    let remote = repo.remote(name, &url.to_string())?;
```